h/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class InvalidUserId(Exception):
```

Comment:
docstring here?

Reply:
Oof. Yes

Reply:
This isn't showing up here, but it is fixed in the diff if you go hunting:

https://github.com/hypothesis/h/pull/5819/files#diff-a88f31737d0a06706739c736f0675905R1